crates/aptos/src/common/types.rs

```diff
@@ -2350,4 +2351,16 @@ pub struct ChunkedPublishOption {
     /// Use this option for publishing large packages exceeding `MAX_PUBLISH_PACKAGE_SIZE`.
     #[clap(long)]
     pub(crate) chunked_publish: bool,
+
+    /// Address of the `large_packages` move module for chunked publishing
+    #[clap(long)]
```

Comment:
Any reason this is Option<String> instead of just String with a default value at the clap level?

Reply:
That will be better, I'll fix it. Thanks!